framework/src/org/checkerframework/common/value/util/Range.java

```diff
@@ -853,26 +886,39 @@ private boolean isWithinInteger() {
      * <p>If the BigInteger range is too wide, i.e., wider than the full range of the Long class,
      * return EVERYTHING.
      *
-     * <p>If the BigInteger bounds are out of the Long type scope, convert the bounds to Long type
-     * in accordance with Java overflow rules, e.g., Long.MAX_VALUE + 1 is converted to
-     * Long.MIN_VALUE.
+     * <p>If the BigInteger bounds are out of the Long type scope and this range does not ignore
+     * overflow, convert the bounds to Long type in accordance with Java overflow rules, e.g.,
+     * Long.MAX_VALUE + 1 is converted to Long.MIN_VALUE.
+     *
+     * <p>If the BigInteger bounds are out of the Long type scope and this range ignores overflow,
```

Comment:
I changed the wording to be more clear. This is the wording the original comment on this method used before I changed it, but I agree that I ought to have made it more precise when editing it.